packages/data/src/redux-store/combine-reducers.js

```diff
@@ -0,0 +1,17 @@
+export function combineReducers( reducers ) {
```

Comment:
Should we add a failproof check to ensure `reducers` is a proper object? Should be cheap IMHO. If you agree we should, it could be yet another case to add a test for.

Alternatively, we could introduce `combineReducers` as a typed function. Then we wouldn't be worried about non-object `reducers` arguments.

Reply:
Neither Redux nor `turbo-combined-reducers` do such a check on `reducers`. Both of them do an `Object.keys( reducers )` call at the very beginning, that's where they crash if it's not a proper object.

`@wordpress/data` re-exports the Redux type for `combineReducers`, so the function has been already typed, and continues to be.

Reply:
Agreed, no need to overcomplicate 👍